src/Pages/License/Search/searchLicense.jsx

```diff
@@ -0,0 +1,209 @@
+      headerName: "Status",
+      width: 75,
+      editable: false,
+      renderCell: (params) =>
+        params.row.status === "Activated" ? <CheckCircleOutlineIcon color="success" /> : <DoNotDisturbIcon color="disabled" />,
+    },
+    {
+      field: "user",
+      headerName: "User",
+      width: 100,
+      editable: false,
+    },
+    {
+      field: "createdOn",
+      headerName: "Created",
+      width: 120,
+      editable: false,
+      renderCell: (params) => formatDateFromMilliSeconds(params.row.createdOn),
+    },
+  ];
+
+  const handleGetRowId = (e) => {
+    return e.licenseId;
+  };
+
+  return (
+    <Grid container xs={6}>
+      <Grid item xs={12}>
+        Input keyworkds to filter the database. Click Search and find the
+        results below.<br></br> You can filter the database for one or more
+        parameters. At least one parameter must be set
+      </Grid>
+      <Grid xs={4} pt={2}>
+        <TextField
+          label="Product Key"
+          variant="outlined"
+          fullWidth
+          name="productKey"
+        />
+      </Grid>
+      <Grid xs={1}></Grid>
+
+      <Grid xs={4} pt={2}>
+        <TextField
+          label="Customer Name"
+          variant="outlined"
+          fullWidth
+          name="customerName"
+        />
+      </Grid>
+      <Grid xs={3}></Grid>
+
+      <Grid xs={4} pt={2}>
+        <TextField
+          label="License Server ID"
+          variant="outlined"
+          fullWidth
+          name="licenseServerId"
+        />
+      </Grid>
+      <Grid xs={1}></Grid>
+
+      <Grid xs={4} pt={2}>
+        <TextField
+          label="Sales Order No."
+          variant="outlined"
+          fullWidth
+          name="orderNo"
+        />
+      </Grid>
+      <Grid xs={3}></Grid>
+
+      <Grid xs={12} marginTop={3}>
+        By default, the most recently created key is shown first.
+      </Grid>
+      <Grid xs={12}>
+        Pick a date to only display keys that where created before a certain
+        date.
+      </Grid>
+      <Grid marginTop={3} xs={3}>
+        <LocalizationProvider dateAdapter={AdapterDateFns}>
+          <Stack spacing={3}>
+            <DatePicker
+              disableFuture
+              label="Only show results before"
+              openTo="day"
+              views={["year", "month", "day"]}
+              value={value}
+              onChange={(newValue) => {
+                setValue(newValue);
+              }}
+              renderInput={(params) => <TextField {...params} />}
+            />
+          </Stack>
+        </LocalizationProvider>
+      </Grid>
+      <Grid xs={4} />
+      <Grid xs={3} alignSelf="self-end">
+        <Box display="flex" justifyContent="flex-end">
+          <Button variant="contained" onClick={loadLicense} label="SEARCH">
+            SEARCH
+          </Button>
+        </Box>
+      </Grid>
+      <Grid xs={12} marginTop={5}>
+        <DataGridCustom
+          setPageSize={setPageSize}
```

Comment:
PageSize should be a local state on DataGridCustom. Changing page size should be handled in the grid component itself

Reply:
done